sunpy/map/header_helper.py

```diff
@@ -127,10 +126,64 @@ def make_fitswcs_header(data,
             dsun_obs = coordinate.observer.radius
         meta_wcs['rsun_obs'] = sun._angular_radius(coordinate.rsun, dsun_obs).to_value(u.arcsec)
 
+    # ---------------------------------------------------------------------
+    # Add support for OBSGEO-[XYZ] FITS header keys for ground-based observers
+    # ---------------------------------------------------------------------
```

Comment:
The original issue mentioned being able to pass in a EarthLocation and that returning  the OBSGEO-[XYZ] triplet in the FITs header.

So I don't understand how this logic fixes that.